phpBB/includes/functions_content.php

```diff
@@ -1248,6 +1236,18 @@ function parse_attachments($forum_id, &$message, &$attachments, &$update_count,
 
 	$unset_tpl = array_unique($unset_tpl);
 
+	// Sort correctly
+	if ($config['display_order'])
+	{
+		// Ascending sort
```

Comment:
ok, probably despite it's descending technically, it's acsending in respect of the attachments order.